agreement-view-component.js

```diff
@@ -0,0 +1,46 @@
+/*!
+ * Copyright (c) 2016 Digital Bazaar, Inc. All rights reserved.
+ */
+define([], function() {
+
+'use strict';
+
+function register(module) {
+  module.component('brAgreementView', {
+    controller: Ctrl,
+    bindings: {
+      group: '@brAgreementGroup',
+      onAgreedRedirectPath: '@brOnAgreedRedirectPath'
```

Comment:
Not sure about this name either.  This is the path to go to after the agreements have been accepted.